setup.py

```diff
@@ -2,11 +2,22 @@
 
 install_requires = ["gitpython", "semantic_version", "btest"]
 
+
+def version():
+    with open("VERSION") as f:
+        return f.read().replace("-", ".dev", 1).strip()
+
+
+def long_description():
+    with open("README") as f:
+        return f.read()
+
+
 setup(
     name="zkg",
-    version=open("VERSION").read().replace("-", ".dev", 1).strip(),
+    version=version(),
```

Comment:
No need to change, just leaving it here as an alternative:

```
pathlib.Path("README").read_text().replace(...)
```

Reply:
I like that more than the manual open calls. I'll still keep the separate functions though since they help reduce complexity in the args of `setup`.